src/patternfly/components/Table/table-th.hbs

```diff
@@ -29,6 +29,9 @@
   {{#if table-th--attribute}}
     {{{table-th--attribute}}}
   {{/if}}>
+  {{#if table-th--IsColumnHelp}}
+    <div class="pf-c-table__column-help">
```

Comment:
should the class names for these be `pf-c-table__sort-column-help` because from what i can see it should only be included inside table sort? that way someone wont add this to anywhere inside the table

Reply:
It's a feature of column headers, not just sortable ones. It's like a "more info" blurb about what the column is for.

Reply:
should it be `pf-c-table__column-header-help` then?

Reply:
It could but that's longer, and it's really just info about the column. It could also be `pf-c-table__th-help` or something like that. Do you see a problem with it named `__column-help`?